dataframely/_base_schema.py

```diff
@@ -137,6 +137,25 @@ def __new__(
                         f"which are not in the schema: {missing_list}."
                     )
 
+        # 3) Check that all members are non-pathological (i.e., user errors).
+        for attr_name, attr_value in namespace.items():
+            if attr_name.startswith("_"):
+                continue  # skip internal names
```

Comment:
This condition seems too wide, I intendedly use the `_` prefix to distinguish some columns

Reply:
This should probably made consistent with [this line](https://github.com/Quantco/dataframely/blob/main/dataframely/_base_schema.py#L161), where we skip if theres a double underscore